### DISCUSSION TITLE
ignore capslock

### DISCUSSION THREAD

**Discussion by oxmj** [ErikReider/SwayOSD#107] [Q&A]
Hi,
please, can someone advise me on how to configure swayosd to ignore capslock? i have Capslock remaped as Ctrl and constant flashing is a bit disconcerting :)
thanks in advance
i am running latest swayosd-git from AUR with hyprland

**Comment by oxmj** [comment]
rebooted and caps lock popup is gone... hmm, no idea what  happened. probably turned it on wiggling with swayosd-client...
anyway, this is solved :)

**Comment by curtrube** [comment]
Hi, I'm running into the same situation. Was there specific configuration to disable the caps lock display?

**Reply by ErikReider** [reply]
There's a `ignore_caps_lock_key` config option for the libinput backend. Other than that, you should just be able to disable and stop the backend through `systemctl disable --now swayosd-libinput-backend`

**Reply by oxmj** [reply]
i have 

empty configs in `/etc/xdg/swayosd` (just sections, all variables commented out), no config in `/etc/libinput`

`swayosd-libinput-backend.service` disabled

in `hyprland.conf` i have this for swayosd
```
# osd for hw buttons (volume, brightness)
exec-once = swayosd-libinput-backend 2>&1 >~/logs/swayosd-libinput-backend.log
exec-once = swayosd-server 2>&1 >~/logs/swayosd-server.log
bindel = , XF86AudioRaiseVolume, exec, swayosd-client --output-volume +3 #--max-volume 120
bindel = , XF86AudioLowerVolume, exec, swayosd-client --output-volume -3 #--max-volume 120
bindl = , XF86AudioMute, exec, swayosd-client --output-volume mute-toggle
bindl = , XF86AudioMicMute, exec, swayosd-client --input-volume mute-toggle
bindle = , XF86MonBrightnessUp, exec, swayosd-client --brightness +3
bindle = , XF86MonBrightnessDown, exec, swayosd-client --brightness -3
```
and this for keyboard setup
```
input {
# KEYBOARD
repeat_rate = 45
repeat_delay = 350
kb_layout = us,cz
# grep -i 'cz' /usr/share/X11/xkb/rules/base.lst
kb_variant = ,qwerty
kb_model =
# man xkeyboard-config
#   CapsLock as a Ctrl
#   alt+alt - toggle language
kb_options = ctrl:nocaps,grp:alts_toggle
kb_rules =
...
```

no issues since then...
HTH
Cheers!
